pkg/injector/patcher/sidecar_container_test.go

```diff
@@ -426,6 +429,7 @@ func TestGetSidecarContainer(t *testing.T) {
 			"--metrics-port", "9090",
 			"--config", "config",
 			"--placement-host-address", "placement:50000",
+			"--scheduler-host-address", "scheduler:50006",
```

Comment:
Same here

Reply:
oh maybe now after changing to the env var. I just already have this change. Ill rm it rq